src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/customer_threads.yml

```diff
@@ -19,7 +38,7 @@ admin_customer_threads_reply:
     customerThreadId: \d+
 
 admin_customer_threads_update_status:
-  path: /{customerThreadId}/update-status/{newStatus}
```

Comment:
I noticed that it doesn't make sense to have status on post path. I hope its okay that i changed how this update works.